effects/disrupter_cannon.lua

```diff
@@ -0,0 +1,100 @@
+-- vulcanfx
+
+return {
+	["disrupter_cannon_muzzle"] = {
+		alwaysvisible         = false,
+		usedefaultexplosions = false,
+		shaft = {
+			air                = true,
+			class              = [[CBitmapMuzzleFlame]],
+			count              = 1,
+			ground             = true,
+			underwater         = 1,
+			water              = true,
+			properties = {
+				colormap           = [[0.9 0.7 0.9 0.01 0.5 0.1 0.8 0.01 0 0 0 0.01]],
```

Comment:
consider either some extra spaces between the phases or ideally something like this that separates them neatly:
```suggestion
				colormap           = [[0.9 0.7 0.9 0.01]]
				                 .. [[ 0.5 0.1 0.8 0.01]]
				                 .. [[ 0.0 0.0 0.0 0.01]],
```

Reply:
Not a single example of the nice staggered thing exists in the codebase and i don't trust CEG to not explode with this